_source/_posts/2022-07-20-express-prisma-tailwind.md

```diff
@@ -0,0 +1,665 @@
+            minutes: 8,
+            calories: 100
+          },
+          {
+            exercise: "Running",
+            amount: 1.2,
+            units: "Miles",
+            date: new Date(2022, 1, 3),
+            minutes: 10,
+            calories: 120
+          },
+          {
+            exercise: "Running",
+            amount: 1.5,
+            units: "Miles",
+            date: new Date(2022, 1, 5),
+            minutes: 12,
+            calories: 150
+          },
+          {
+            exercise: "Heavy Bag",
+            amount: 4,
+            units: "Rounds",
+            date: new Date(2022, 1, 1),
+            minutes: 15,
+            calories: 100
+          },
+          {
+            exercise: "Heavy Bag",
+            amount: 6,
+            units: "Rounds",
+            date: new Date(2022, 1, 3),
+            minutes: 22,
+            calories: 150
+          },
+          {
+            exercise: "Heavy Bag",
+            amount: 4,
+            units: "Rounds",
+            date: new Date(2022, 1, 5),
+            minutes: 15,
+            calories: 100
+          },
+          {
+            exercise: "Situps",
+            amount: 50,
+            units: "Reps",
+            date: new Date(2022, 1, 2),
+            minutes: 5,
+            calories: 50
+          },
+          {
+            exercise: "Pushups",
+            amount: 100,
+            units: "Reps",
+            date: new Date(2022, 1, 2),
+            minutes: 10,
+            calories: 100
+          },
+          {
+            exercise: "Situps",
+            amount: 50,
+            units: "Reps",
+            date: new Date(2022, 1, 4),
+            minutes: 5,
+            calories: 50
+          },
+          {
+            exercise: "Pushups",
+            amount: 100,
+            units: "Reps",
+            date: new Date(2022, 1, 4),
+            minutes: 10,
+            calories: 100
+          },
+        ],
+      },
+    },
+  });
+}
+
+main()
+  .catch((e) => {
+    console.error(e);
+    process.exit(1);
+  })
+  .finally(async () => {
+    await prisma.$disconnect();
+  });
+```
+
+Prisma knows to use the `seed.js` file from the command you added to `package.json`.  This file will insert the data when you run that command.  
+
+You can now add and apply the migration using the Prisma CLI. From the root of your application directory, run the following command.
+
+```console
+npx prisma migrate dev --name init
+```
+
+The CLI has several ways to add and apply migrations.  I suggest you understand the best way to manage migrations for your environment.  The method above is the easiest and fastest way to prepare your database.  As part of the migration process, this command will look for the `seed` command from your `package.json` and run that as well.  Once complete, your database should be ready with a database full of seed data ready to work.
+
+### Complete the development of your express application
```

Comment:
I don't know if level 3 headings target keywords, but consider using "Add OIDC authentication" or something like that.  (also consider making it level 2 heading along with previous heading with the word "Prisma" in it??)